database/rrdset.c

```diff
@@ -1089,6 +1091,64 @@ inline void rrdset_next_usec(RRDSET *st, usec_t microseconds) {
 // ----------------------------------------------------------------------------
 // RRDSET - process the collected values for all dimensions of a chart
 
+void rrdset_dump_debug_state(RRDSET *st) {
+#ifdef NETDATA_INTERNAL_CHECKS
+    if (debug_flags&D_REPLICATION) {
```

Comment:
please, add a space between the variables and the bitwise operator to be more readable.

Reply:
Fixed